app/components/primer/alpha/select_panel.rb

```diff
@@ -429,6 +409,12 @@ def initialize(
         @dynamic_aria_label_prefix = dynamic_aria_label_prefix
         @loading_label = loading_label
         @loading_description_id = nil
+
+        @form_builder = form_arguments[:builder]
+        @default_value = form_arguments[:default_value]  
+        @default_values = form_arguments[:default_values]  
```

Comment:
@khiga8 I was using `default_value` for single-select and `default_values` for multi-select but maybe we can just always use `default_values` instead of two props (i.e. pass an array no matter what).... I think it depends on if we keep the same name for all multi-select inputs [(see this comment for more information)](https://github.com/primer/view_components/pull/3340/files#r1968206977)

Reply:
I think that for most scenarios, we can expect the name to remain the same.
 
But in the one EMU example we saw,  the name AND value changes depending on selection, so I fear there are similar edge cases for multi-select that we need to support 🤔 . 

Is there a code search we can do in dotcom to verify?